### PR TITLE
[W5.7][W14-3]Zhang Yuntong

### PR DESCRIPTION
**Pull request by yuntongzhang**

### CODE REVIEW

**Comment by darienchong** [comment]
[W5.7] Team component

Code looks clean.
Might want to consider removing global import for commands.
Might want to consider unit tests for the count command (are they needed?)

-Darien

**Comment by jeffypie369** [comment]
[W5.7] Team component

Everything looks good!
Simple and useful addition

-Jefferson

File: test/java/seedu/addressbook/parser/ParserTest.java [hunk 1]
```diff
@@ -12,17 +12,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (kexiaowen) [inline test/java/seedu/addressbook/parser/ParserTest.java:15]: Do not import *. As stated in our coding standard, imported classes should always be listed explicitly.

**Review by kexiaowen — APPROVED** [review]
@yuntongzhang Nice attempt. Try to add a description for your future PRs so reviewers can find out what your PR does quickly. It is good that you updated relevant documentation and I/O tests after adding the count command. How about unit tests? Pls close this PR after reading my comments.

**Pull request closed without merging**